Sources/GameMP/Computer.cpp

```diff
@@ -246,7 +246,7 @@ void LastUnreadMessage(void)
 // go to next/previous message
 void PrevMessage(void)
 {
-  if (_iActiveMessage<_acmMessages.Count()==0) {
+  if (_iActiveMessage < _acmMessages.Count()==0) { // FIXME: DG: what is this about?
```

Comment:
I think so, yes.

My original confusion when adding the FIXME to the code came from assuming that it was supposed to mean `if(_iActiveMessage < (_acmMessages.Count()==0))`, but `>` has higher priority than `==` so that assumption was incorrect